src/Storage/ABS/SnowflakeImportAdapter.php

```diff
@@ -30,10 +32,12 @@ public function __construct(SourceInterface $source)
 
     /**
      * @inheritDoc
+     * @param Table $destination
      */
     public function executeCopyCommands(
         array $commands,
         Connection $connection,
+        DestinationInterface $destination,
```

Comment:
Tady jsem se trochu zamotal u té abstrakce a pamatuju si že jsem i minule nad tím trochu dumal.
Tady se v obou tech variantách vlastně počítá že to dostane správně připravené ty copy commandy. Tady se např destination vůbec nepoužije a v druhé variantě pro import z tabulky jo.
Chápu že to je kvůli tomu rozdílnému zpracování resultů v obou variantách kvůli BC a jinak by stačilo mít `getCopyCommands`.
Nevím jak to udělat líp tak jenom poznamenávám:)

Reply:
Jo je to presně kvůli tomu zpracování výsledků.
Když sa na to tak dívám tak by to šlo hodit do prdele. A zavolat ten ten select count co je tady: https://github.com/keboola/php-db-import-export/blob/bb6ee90e970fc62d092df0e31200bec1e505bc5c/src/Storage/Snowflake/SnowflakeImportAdapter.php#L42-L46
Je to jedna query navíc, to už nám nic neudělá a zjednoduší to tu implementaci.

Reply:
Vykopl sem to dal tam vlastní `fetchAll`